src/hooks/useBridgesApi/useCanIDeleteBridge.ts

```diff
@@ -0,0 +1,38 @@
+import { useCallback, useState } from "react";
+import { Configuration, ProcessorsApi } from "@openapi/generated";
+import axios from "axios";
+import { useSmartEvents } from "@contexts/SmartEventsContext";
+
+export function useCanIDeleteBridge(): {
+  canIDeleteBridge: (bridgeId: string) => Promise<boolean>;
+  error: unknown;
+} {
+  const [error, setError] = useState<unknown>();
+  const { getToken, apiBaseUrl } = useSmartEvents();
+
+  const canIDeleteBridge = useCallback(
+    (bridgeId: string) => {
+      setError(undefined);
+      const processorsApi = new ProcessorsApi(
+        new Configuration({
+          accessToken: getToken,
+          basePath: apiBaseUrl,
+        })
+      );
+      return processorsApi
+        .listProcessors(bridgeId, undefined, 0, 1)
+        .then((response) => {
+          return response.data?.total === 0;
+        })
+        .catch((err) => {
+          if (!axios.isCancel(err)) {
+            setError(err);
+          }
+          return false;
+        });
```

Comment:
Do we need this check here?

Reply:
I think we can get rid of this check. Thank you @kelvah